chains/evm/contracts/offRamp/CCVAggregator.sol

```diff
@@ -65,22 +68,25 @@ contract CCVAggregator is ITypeAndVersion, Ownable2StepMsgSender {
     address tokenAdminRegistry; // Token admin registry address
   }
 
-  // TODO actually set this config
   /// @dev Per-chain source config (defining a lane from a Source Chain -> Dest OffRamp).
   struct SourceChainConfig {
     IRouter router; // ─╮ Local router to use for messages coming from this source chain.
     bool isEnabled; // ─╯ Flag whether the source chain is enabled or not.
-    address defaultCCV; // Default CCV to use for messages from this source chain.
-    address requiredCCV; // Required CCV to use for all messages from this source chain.
+    bytes onRamp; // OnRamp address on the source chain.
+    address[] defaultCCV; // Default CCV to use for messages from this source chain.
+    address[] laneMandatedCCVs; // Required CCV to use for all messages from this source chain.
```

Comment:
Outlining the relationship between these two in a comment would be useful when reading the code because I'm already mixing up between default and lane mandated (formerly required). My understanding is that `laneMandatedCCV` is a subset of `defaultCCV`, i.e. lane mandated must always be specified and therefore the default set must include them if a user doesn't specify any?